### DISCUSSION TITLE
Recommandations SEO

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2610] [General]
> Je copie ici les travaux de Clément Chapalain et Mathilde Petit sur le référencement de mon-entreprise

# Contexte

mon-entreprise.fr est le guide pratique pour répondre à toutes les questions que l’on peut se poser lorsqu’on souhaite monter et gérer sa société. Le produit était à la base un calculateur de salaire brut / net, intégré à de nombreux sites officiels. Le service s’est ensuite étoffé afin de répondre à un maximum de questions relatives à la création et la gestion des entreprises.

Le calculateur de salaire brut / net a reçu 1,7 million de visites en 2019, et près de 350k en mars 2020. Cependant, seulement 10% des visites sur le site mon-entreprise.fr sont issues des moteurs de recherche. Il y a donc un potentiel de visiteurs importants à gagner en améliorant le référencement du site en général et de cette page en particulier.

Plus d’informations : 

*   [Fiche produit](https://beta.gouv.fr/startups/mon-entreprise.html)
*   [Statistiques d’usage](https://mon-entreprise.fr/stats)

# Références

[Guide sur les bonnes pratiques du SEO](https://beta-gouv.gitbook.io/guide-se/marketing/seo-1)

Guide pratique écrit par Mathilde qui résume toutes les pratiques de base du SEO.

[Google : bien débuter en référencement naturel (SEO)](https://support.google.com/webmasters/answer/7451184?hl=fr)

Le guide de Google pour avoir un site bien référencé, et comprendre comment utiliser les outils Google associés au référencement (Analytics, Search Console, Adwords).

[The Google SEO Bible: Everything a Startup Needs to Know (anglais)](https://salon.thefamily.co/the-google-seo-bible-everything-a-startup-needs-to-know-a60dbac2d060)

Un article de The Family en anglais qui résume ce qui est important pour Google et donne des règles pratiques simples à suivre.

# 

# Recommandations pratiques

## Pour chaque page

Les exemples sont pris sur la page [Calcul du salaire brut / net : le simulateur de l'Urssaf](https://mon-entreprise.fr/simulateurs/salari%C3%A9) mais les recommandations s’appliquent (pour la plupart) à toutes les pages du site.

**URL**

L’url doit être courte, proche de la page racine du site, et contenir les mots-clés utilisés par les usagers. Quelques propositions :

1. Viser les termes de recherche importants, notamment “salaire brut net”, qui est beaucoup plus recherché que “simulateur embauche”, “calculateur salaire”, ou “simulateur salaire” :  \

![pasted image 0](https://user-images.githubusercontent.com/1775934/78873778-ae157880-7a4b-11ea-9153-a5a4f5621665.png)

2. Ne mettre que les mots importants et dans le bon ordre, le premier mot de l’url aura plus d’importance pour google que le deuxième, le deuxième plus que le troisième, etc.

L’URL pourrait par exemple devenir : _mon-entreprise.fr/simulateurs/salaire-brut-net_.

**Page title**

Le page title est la balise que va utiliser Google pour indiquer le titre de la page sur ses résultats de recherche, ici “Calcul du salaire [...]” : 

![pasted image 0](https://user-images.githubusercontent.com/1775934/78873844-c6859300-7a4b-11ea-8a8a-2781443a65d5.png)

Quelques recommandations : 
![pasted image 0](https://user-images.githubusercontent.com/1775934/78873959-f03eba00-7a4b-11ea-8871-05a016a7be87.png)

*   Jamais plus de 70 caractères, sinon ce sera coupé dans le titre affiché par Google et le résultat baissera dans les résultats de recherche !
*   Utiliser les mots-clés les plus recherchés : simulateur → calculateur
*   Le premier mot a plus de poids que le deuxième, qui a plus de poids que le troisième, etc. → enlever “Calcul du” dans le titre
*   Utiliser des “ - ” plutôt que des “ | ” pour les séparations

**Page description**

Le page description est ce que va utiliser Google comme texte descriptif sous votre titre dans les résultats de recherche, ici “Calculer son salaire net [...]”. C’est souvent ce qui est contenu dans la balise description, mais Google peut décider d’utiliser une autre partie de la comme description s’il considère que cela répond mieux à la requête de l’utilisateur.

![pasted image 0](https://user-images.githubusercontent.com/1775934/78873981-f7fe5e80-7a4b-11ea-92c5-fc18f1ba9a96.png)

Quelques recommandations : 

*   65-80 caractères maximum, afin qu’elle ne soit pas coupée et qu’elle ait plus de chance d’être utilisée par Google
*   Faire des phrases
*   Utiliser des synonymes des mots-clés de la balise title, et reprendre les mots les plus importants (salaire, brut, net, calcul)

**Balises open graph**

Les balises open graph n’ont a priori pas d’incidence directe sur le référencement, mais il est recommandé de les définir correctement pour améliorer le partage sur les réseaux sociaux et ainsi favoriser les visites depuis d’autres sites.

Sur mon-entreprise, les balises ne sont pas personnalisées par page et peuvent donc induire en erreur l’utilisateur qui se rend sur le calculateur. [Cet outil de facebook](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fmon-entreprise.fr%2Fsimulateurs%2Fsalari%25C3%25A9) permet de vérifier le rendu du partage. [Cet outil de Google](https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fmon-entreprise.fr%2Fsimulateurs%2Fsalari%25C3%25A9) permet également de vérifier simplement toutes les données structurées ajoutées à une page.

**H1, H2, H3**

Quelques recommandations : 

*   Ne jamais dépasser 80 caractères pour les H1, H2, H3
*   H1 doit être différent du titre de la page (balise title) tout en utilisant les mots-clés importants, par exemple le titre de la page pourrait être “Salaire brut / net : calculateur de l’Urssaf” et le H1 “Calculez votre salaire brut / net mensuel et annuel”
*   H2 doit être une précision de H1, potentiellement une question si cela s’y prête : “Comment calculer son salaire net ?” et “Comment calculer le coût d’embauche ?” par exemple

**Page speed**

La vitesse de chargement est une donnée a priori de plus en plus importante pour Google. [L’outil de test de Google](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fmon-entreprise.fr%2Fsimulateurs%2Fsalari%25C3%25A9) donne une note de 61/100 au calculateur. 

Je vous laisse estimer la faisabilité des actions recommandées ! Google indique qu’elles n’ont pas d’incidence “directe” dans la performance de vos pages dans son algorithme de recherche mais beaucoup de spécialistes recommandent de tout de même l’optimiser. Pour info les autres résultats avant le mon-entreprise.fr sur la requête “salaire brut net” ont une note de vitesse de chargement de : 29, 78, 33, 24, 24.

**Contenu**

Utiliser les outils suivants pour savoir quels sont les thèmes à aborder dans le contenu de la page : 

*   [Ubersuggest](https://app.neilpatel.com/fr/ubersuggest/overview?lang=fr&locId=2250&keyword=salaire+brut+net) : obtenir des suggestions de mots-clés, de pages, de titres, etc.
*   [Google trends](https://trends.google.fr/trends/?geo=FR) : comparer la popularité de mots-clés sur Google
*   [Answer the public](https://answerthepublic.com/) : trouver des idées de mots-clés à utiliser
*   Cliquer sur le résultat des sites sur Google et ensuite revenir à la recherche en faisant précédent, Google vous proposera alors les autres recherches pertinentes sur ces mots-clés : 

![pasted image 0](https://user-images.githubusercontent.com/1775934/78874062-119fa600-7a4c-11ea-9dd0-d16e85459276.png)

Globalement, il faut essayer de ne pas trop “stacker” des mot-clés mais plutôt essayer de répondre aux questions que se posent réellement les gens sous forme de FAQ, avec des mots-clés différents à chaque fois.

## Arborescence et navigation

Pour améliorer la lisibilité d’un site pour les moteurs de recherche, il est important qu’il soit bien structuré et lisible par les moteurs de recherche.

**Navigation**

La page d’accueil et les pages de niveau 1 (mon-entreprise.fr/xxx) doivent être accessibles depuis toutes les pages, via une navigation, un menu, des liens dans le footer, etc.

**Arborescence des URLs**

Une page parent doit permettre d’accéder à toutes ses pages enfants, et ne permet si possible pas de naviguer vers une autre partie du site.

Par exemple, cette page [mon-entreprise.fr/créer/auto-entrepreneur](https://mon-entreprise.fr/cr%C3%A9er/auto-entrepreneur) possède trois liens principaux : 

*   1 vers une page /simulateurs/auto-entrepreneur
*   1 vers une page /créer/créer/après-la-création
*   1 vers une ressource externe

Dans l’idéal, les liens principaux de cette page devraient rediriger vers les différentes pages en /créer/auto-entrepreneur/xxx.

**Toutes les pages doivent être accessibles depuis la page d’accueil**

J’ai cru comprendre que vous aviez beaucoup de pages de documentation (par exemple celle-ci : [mon-entreprise.fr/documentation/contrat-salarié/rémunération/net-après-impôts](https://mon-entreprise.fr/documentation/contrat-salari%C3%A9/r%C3%A9mun%C3%A9ration/net-apr%C3%A8s-imp%C3%B4t)), mais elle ne semblent pas référencées sur le site. Il n’y a pas de lien vers la page mon-entreprise.fr/documentation sur la page d’accueil, ce qui permettrait aux crawlers des moteurs de recherche de mieux référencer ces pages et de mieux “comprendre” l’organisation du site.

## Backlinks

**Partenariats avec d’autres site**

Mon-entreprise n’a que peu de backlinks provenant d’autres sites (12 297, cf. capture d’écran de gauche), en comparaison par exemple avec le Code du travail numérique (243 511, cf. capture d’écran de droite) :

Des partenariats avec des sites publics pour intégrer des liens vers mon entreprise et notamment sa documentation pourraient être envisagés afin d’augmenter le nombre de liens externes pointant vers mon-entreprise.

**Intégrations des simulateurs**

Les simulateurs intègrent un lien vers le site mon-entreprise.fr, mais il est très peu visible et derrière une image, ce qui n’encourage pas l’utilisation de ces _backlinks_.

Par exemple, sur le site du Code du travail numérique, il n’y a eu que 1 clic sur le lien mon-entreprise en mars 2020, sur les 4 889 vues du simulateur, soit un taux de conversion de 0,02%.

Quelques recommandations :

*   Ajouter un lien textuel vers le site mon-entreprise.fr qui soit plus visible des utilisateurs
*   Ajouter des liens vers d’autres simulateurs ou d’autres ressources intéressantes (comme ce qui a commencé à être fait avec le simulateur d’activité partielle sur le simulateur de salaire brut net par exemple)

## Sitemap

La sitemap sur Google Search Console n’est pas à jour, elle ne comporte que 34 pages différentes :

En donnant toutes les pages à Google via le sitemap, ses crawlers découvriront plus rapidement l’exhaustivité du site.

**Comment by mquandalle** [comment]
La plupart des éléments ont été corrigés dans #1085, j'ajoute quelques idées en vrac ici :

- [ ] Utiliser des données structurées pour les questions dans les "textes SEO" https://developers.google.com/search/docs/data-types/qapage?hl=fr
- [ ] Une FAQ ? Ou en tout cas identifier les questions (posées sous formes de questions sur Google) auxquelles on répond mais pour lesquelles on n'est pas référencés
- [ ] mieux référencer les pages « A VS B » pour la comparaison de statuts et trouver les A et B qui sont recherchés (on parle par exemple « d'entreprise individuelle » qui n'est pas recherché)

On est en train de collecter les mots clés pertinents

**Comment by laem** [comment]
J'ajoute que #1411 introduit en meta les titres et descriptions des centaines de pages de la doc, qui peuvent du coup être mieux comprises par google et donc lui faire apparaître beaucoup de contenu :)
